src/test/java/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,85 @@
+package service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import core.basesyntax.service.UserException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
+    private User standartUser;
```

Comment:
You can init user and reuse it with some changes for every needing case, or clear this field